README.md

```diff
@@ -102,7 +102,7 @@ If you are using Maven, you need to add the following dependency:
 
 If you are using Gradle, here is the dependency to add:
 
-`compile group: 'com.yoti', name: 'yoti-sdk-impl', version: '1.3'`
+`compile group: 'com.yoti', name: 'yoti-sdk-impl', version: '1.4.1'`
```

Comment:
we are re-updating this once we release the fix, right?

Reply:
Yes.  As soon as 1.4.2 is out there, I will update the readme.